rollup/cmd/rollup_relayer/app/app.go

```diff
@@ -121,6 +131,19 @@ func action(ctx *cli.Context) error {
 	return nil
 }
 
+func readGenesis(genesisPath string) (*core.Genesis, error) {
+	file, err := os.Open(filepath.Clean(genesisPath))
+	if err != nil {
```

Comment:
How about move this logic to [here](https://github.com/scroll-tech/scroll/tree/develop/rollup/internal/config)